server/views/resources/shop.jade

```diff
@@ -4,10 +4,13 @@ block content
     hr
     .row
         .col-xs-12.col-sm-10.col-sm-offset-1.col-md-8.col-md-offset-2.text-center
-            img.img-responsive.img-center(src='//i.imgur.com/FNFqn9Tg.jpg')
-            h3 Free Code Camp laptop stickers
-            h4 Get 2 for only $5, with free shipping anywhere!
-            p These durable 2" (5 cm) stickers sport a matte finish, and look great anywhere - especially on your laptop.
+          <form action="//www.paypal.com/cgi-bin/webscr" onSubmit="ga('send', 'event', 'Shop', 'clicked', 'Freecodecamp sticker');" method="post" target="_blank">
```

Comment:
I think for now it's fine as long as it works. Where you able to verify this PR?